### DISCUSSION TITLE
How to delete local cache after changing cache directory

### DISCUSSION THREAD

**Discussion by skrishnanv** [treeverse/dvc#7015] [Help] [answered]
Hi,
I am experimenting with dvc to see how to share data repo and cache. as a first step:
1. i added an external data path using dvc add --external  from say /mnt/d/datapath. This resulted in a local cache in .dvc/cache in some path inside  /mnt/c where my project lies.
2. then i added a external cache in /mnt/d/cache and repeated the command. i checked that the cache is created and data is copied.
3. i added a remote in /mntd/dvcremote and did a dvc push. i saw that now the data is pushed here in this location and is copied as well.
4. Now i want to delete the cache which is in my local .dvc. Can anyone let me know how i could do it? i tried dvc gc -w but it did not remove anything.
Thanks

**Comment by pmrowla** [comment, marked as answer]
If you are certain that everything has been pushed and you no longer need the local data, you can just delete the entire local cache directory
```
rm -rf .dvc/cache
```

**Reply by skrishnanv** [reply]
thanks, i did think of that but I was not sure if it would mess up things. However, i would expect that the gc would do this.

**Reply by pmrowla** [reply]
@skrishnanv `gc` works by specifying what you want to preserve, so `gc -w` will delete everything from cache _except_ for the files which are used by your current workspace.

When you want to remove everything, the normal way is to just use `rm`
